locale/hu/LC_MESSAGES/django.po

```diff
@@ -1685,7 +1685,7 @@ msgid "An author has been added to your add-on"
 msgstr "A kiegészítőhöz egy szerző hozzárendelésre került"
 
 msgid "Author invitation for {addon_name}"
-msgstr "Szerzői meghívás a(z) {addon} kiegészítőhöz."
+msgstr ""
```

Comment:
you can change it to `Szerzői meghívás a(z) {addon_name} kiegészítőhöz.` so it's not broken.

Reply:
Are you sure this is the right translation? I don't speak Hungarian... I feel like letting localizers update this string would be safer but 🤷‍♂

Reply:
I don't, but I wasn't updating the translation from what was written, it was just correcting the broken `{addon_name}` token.

Reply:
I see, but why is that better than letting localizers update the entire localized string? In the frontend, I think we've seen such issues where the localized string was completely different (not only a problem with the placeholder name but with the whole string)

Reply:
When there is only a single placeholder, and it looks like an obvious typo in the placeholder name, we tend to fix it ourselves, it's just faster.